rails/test/lib/normalize.rb

```diff
@@ -0,0 +1,15 @@
+require 'active_support'
+
+# Taken from https://stackoverflow.com/questions/7275952/how-can-i-sort-yaml-files/13368706#13368706
+
+module Normalize
+  class << self
+    def deep_sort(object)
+      return object unless object.is_a?(Hash)
+      hash = RUBY_VERSION >= '1.9' ? Hash.new : ActiveSupport::OrderedHash.new
+      object.each { |k, v| hash[k] = deep_sort(v) }
+      sorted = hash.sort { |a, b| a[0].to_s <=> b[0].to_s }
```

Comment:
Can you check for the keys `zero`, `one`, `two`, `few`, `many`, `other` and make sure they always appear before all other key and in that order, but only if there are no other keys present? I would much prefer to keep the logical order for pluralization rules.

e.g.
``` ruby
PLURALIZATION_KEYS = %w( zero one two few many other ).freeze
# ...
      sorted = if (hash.keys.map(&:to_s) - PLURALIZATION_KEYS).empty?
        hash.sort { |a, b| PLURALIZATION_KEYS.index(a[0].to_s) <=> PLURALIZATION_KEYS.index(b[0].to_s) }
      else
        hash.sort { |a, b| a[0].to_s <=> b[0].to_s }
      end
```

Reply:
I think it's reasonable to have an exception in that case. Will do!